modules/cluster-logging-loki-about.adoc

```diff
@@ -0,0 +1,13 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: CONCEPT
+[id="about-logging-loki_{context}"]
+= About Loki
+
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system currently offered as an alternative to Elasticsearch as a log store for the {logging}.
+
+Elasticsearch indexes incoming log records completely during ingestion. Loki only indexes a few fixed labels during ingestion, and defers more complex parsing until after the logs have been stored. This means Loki can collect logs more quickly. As with Elasticsearch, you can query Loki using JSON paths or regular expressions.
+
```

Comment:
As with Elasticsearch, you can query Loki using JSON paths or regular expressions. Could we add a link https://grafana.com/docs/loki/latest/logql/ here?

Reply:
You mean as an embedded link instead of under an additional resources section? [Like this?](https://grafana.com/docs/loki/latest/logql/)